src/azure-firewall/azext_firewall/_validators.py

```diff
@@ -143,3 +143,15 @@ def validate_rule_group_collection(namespace):
     if namespace.target_fqdns is not None and namespace.fqdn_tags is not None:
         raise CLIError('usage error: --target-fqdns | --fqdn-tags')
     return namespace
+
+
+def validate_private_ranges(namespace):
+    namespace.private_ranges = ', '.join(namespace.private_ranges)
```

Comment:
No return?

Reply:
no return. If you don't need parameter in validator, you don't need return anything. Usually you need to return a function if this validator need parameters.